deploy/operator/hypershift/playbooks/templates/hasc.j2

```diff
@@ -0,0 +1,26 @@
+apiVersion: agent-install.openshift.io/v1beta1
+kind: HypershiftAgentServiceConfig
+metadata:
+ name: hypershift-agent
+ namespace: {{ spoke_namespace }}
+spec:
+ kubeconfigSecretRef:
+   name: {{ spoke_kubeconfig_secret }}
+ databaseStorage:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+   requests:
+    storage: 8Gi
+ filesystemStorage:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+   requests:
+    storage: 8Gi
+ imageStorage:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+   requests:
+    storage: 10Gi
```

Comment:
nit: always let your files end with a line-break character, to make it easier for Linux CLI tools

Reply:
yep, copy-paste mistake:/